### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by vladsavelyev**
Initial documentation: intro to workflows, seqr-loader diagram, jobs/stages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Hail Batch Workflows
+
+This repository provides implementations of genomics workflows using Hail Batch and the [cpg_utils/workflows library](https://github.com/populationgenomics/cpg-utils/blob/main/cpg_utils/workflows/README.md), specifically:
+
+* WES/WGS Seqr Loader: FASTQ -> CRAM -> GVCF -> pVCF -> Matrix Table -> Elasticsearch index, with an ability to use parts of this pipeline as e.g. a single-sample germline variant calling workflow (FASTQ -> GVCF), joint-calling pipeline (GVCF -> pVCF), AS-VQSR pipeline, etc.
+* GATK-SV: CRAM -> VCF and more, orchestrates [GATK-SV WDL workflows](https://github.com/broadinstitute/gatk-sv) in Hail Batch.
+
+## Seqr Loader
+
+![uml](docs/seqr_loader.png)
```
Comment (lgruen) [inline README.md:10]: From the diagram it looks like all the Hail Query stages are run through Dataproc. Is that correct? The last stage (writing to Elasticsearch) probably has a Spark dependency, but couldn't we run everything else in Query on Batch?

Do the pedigree checks actually happen twice?

Do you also want to add an editable version of this diagram into this repository?

**Review by lgruen — CHANGES_REQUESTED** [review]
This is awesome! Thanks for writing this up.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Seqr Loader is a combination of the following [WARP](https://github.com/broadinstitute/warp) workflows:
+
+* Single-sample whole genome and WES germline calling, FASTQ -> GVCF (BWA/DRAGMAP, GATK4 HaplotypeCaller)
+* Re-alignment, CRAM -> GVCF (we use [BAZAM](https://github.com/ssadedin/bazam) to extract FASTQ from CRAM/BAM)
+* Whole genome and WES joint-calling, GVCFs -> VCF
+* WES AS-VQSR and WGS VQSR workflows (we adjusted to use AS-VQSR for WGS as well)
```
[the remark this answers is not in the document]
Comment (vladsavelyev) [reply README.md:17]: Just noting that it wasn't available in WARP for WGS

File: README.md (hunk 1 above)
Comment (vladsavelyev) [reply README.md:10]: > From the diagram it looks like all the Hail Query stages are run through Dataproc. Is that correct? The last stage (writing to Elasticsearch) probably has a Spark dependency, but couldn't we run everything else in Query on Batch?

We have that query code ready to run on Batch backend, we tested that, and they can be quickly swapped to use Batch. But it was tricky to debug issues with Batch backend, so I switched to a more stable Spark backend for this production code. We can switch back in the future.

Comment (vladsavelyev) [reply README.md:10]: > Do you also want to add an editable version of this diagram into this repository?

Added a Miro board URL

File: README.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+This repository provides implementations of genomics workflows using Hail Batch and the [cpg_utils/workflows library](https://github.com/populationgenomics/cpg-utils/blob/main/cpg_utils/workflows/README.md), specifically:
+
+* WES/WGS Seqr Loader: FASTQ -> CRAM -> GVCF -> pVCF -> Matrix Table -> Elasticsearch index, with an ability to use parts of this pipeline as e.g. a single-sample germline variant calling workflow (FASTQ -> GVCF), joint-calling pipeline (GVCF -> pVCF), AS-VQSR pipeline, etc.
+* (in progress) GATK-SV: CRAM -> VCF and more, orchestrates [GATK-SV WDL workflows](https://github.com/broadinstitute/gatk-sv) in Hail Batch.
+
+## Seqr Loader
+
+![uml](docs/seqr_loader.png)
+
```
Comment (lgruen) [inline README.md:11]:
```suggestion
(See [this Miro board](https://miro.com/app/board/uXjVPapXnfM=/) to update the graph above.)

```

File: docs/README [hunk 4]
```diff
@@ -0,0 +1 @@
+The graph is designed in this Miro board: https://miro.com/app/board/uXjVPapXnfM=/
```
Comment (lgruen) [inline docs/README:1]: I'd suggest putting that inline in the doc (see above), so there's no confusion between `README.md` and `README` in the same directory.

File: README.md (hunk 1 above)
Comment (lgruen) [reply README.md:10]: Thank you! Just the one remaining question then, about the pedigree checks being shown twice in the diagram.

**Review by lgruen — APPROVED** [review]

**Pull request merged**